pandas/_libs/groupby.pyx

```diff
@@ -714,10 +714,12 @@ group_ohlc_float64 = _group_ohlc['double']
 
 @cython.boundscheck(False)
 @cython.wraparound(False)
-def group_quantile(ndarray[float64_t] out,
-                   ndarray[int64_t] labels,
-                   numeric[:] values,
-                   ndarray[uint8_t] mask,
+def group_quantile(floating[:, :] out,
+                   int64_t[:] counts,
+                   floating[:, :] values,
+                   const int64_t[:] labels,
+                   Py_ssize_t min_count,
```

Comment:
The new arguments are unused right? I think OK to name them as such then too (unless Cython supports the Py_UNUSED macro; not sure it does)

Reply:
Only min_count is unused. And in rewriting _get_cythonized_result, I realized that it doesn't even need to be there. So all arguments are now used.